File: PrimeNumber.cpp
/*
Print the first K prime number
*/

#include <iostream>
#include <cmath>
using namespace std;

void PrimeNum(int k)
{
    if(k<=0)
    return;
    int num=2;
    int i,j;
    for(i=1;i<=k;)
    {
        for(j=2;j<=sqrt(num);j++)
        {
            if(num%j==0)
            break;
        }
        if(j>sqrt(num))
        {
            cout<<num<<endl;
            i++;
        }
        num++;
    }
}
int main()
{    
   int K=10;
   PrimeNum(K);
   return 0;
}

File: PrimeNumber.py
'''
Print the top K prime number
'''

import math

def PrimeNumber(K):
    i=1
    num=2
    while i<=K:
        r = int(math.sqrt(num))
        for j in range(2,r+1): #[m+2 for m in range(r)]:
            if num%j==0:
             break;
        if j==r+1:
           print num
           i +=1
        num +=1   


if __name__=="__main__":
    K = 10
    PrimeNumber(K)
    
